README.md

```diff
@@ -1,13 +1,10 @@
-# SangeFault
+# <img src="https://raw.githubusercontent.com/SangeloDev/SangeFault/1.8.9/pack.png" width="32"> SangeFault
 
-[![UC2Badge](https://img.shields.io/badge/UC%C2%B2-supported-%23FE7D37)](https://uc.gamingcraft.de) [![LabymodBadge](https://img.shields.io/badge/LabyMod-supported-%23008FE8)](https://labymod.net)
-
-[![PMC](https://img.shields.io/badge/PlanetMinecraft-visit-%236EC310)](https://www.planetminecraft.com/texture-pack/sangefault-1-8-x-resource-pack/) [![Website](https://img.shields.io/badge/Coming%20Soon-pick%20and%20choose-blue)](https://pack.sangelo.space)
-
-ℹ️ If you're viewing this page from [Gitea](https://git.sangelo.space/Sangelo/SangeFault) and don't see any releases, please view them on [GitHub](https://github.com/SangeloDev/SangeFault/releases).
+[![](https://img.shields.io/badge/UtilityClient-supported-%23aa4465?style=for-the-badge)](https://uc.gamingcraft.de) [![](https://img.shields.io/badge/LabyMod-supported-%23008FE8?style=for-the-badge)](https://labymod.net)

```

Comment:
I'd prefer the old badges to stay, but all in all this PR is alright.

Reply:
they're ugly and don't fit your design. Won't fix.